### PR TITLE
[FIX] account_interest: Fix to avoid the unnecessary recomputation of  l10n_latam_document_type_id

### PR DESCRIPTION
**Pull request by rov-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-financial-tools/pull/570).

File: account_interests/models/res_company_interest.py [hunk 1]
```diff
@@ -200,7 +200,8 @@ def create_invoices(self, to_date, groupby='partner_id'):
             journal = self.env['account.move'].with_context(
                 internal_type='debit_note', default_move_type='out_invoice').new(
                     {'partner_id': partner_id, 'move_type': 'out_invoice'}).journal_id
```
Comment (maq-adhoc) [inline account_interests/models/res_company_interest.py:202]: Ya que estamos, ¿no mejoraría esto usar un with_company, o otro método para asegurarnos que el ejecute el new en la compañia correcta?

**Comment by maq-adhoc** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@rov-adhoc @maq-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/account-financial-tools#574

**Comment by fw-bot-adhoc** [comment]
@rov-adhoc @maq-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/account-financial-tools#574

**Comment by fw-bot-adhoc** [comment]
@rov-adhoc @maq-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/account-financial-tools#574

**Pull request closed without merging**